curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f1.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-41
 
 # --description--
 
-You want to add the same gradient to the next two sections. Instead of doing that, create a new class called `bb1-window`, and move the `height` and `background` properties and values from `.bb1a` to the new class.
+You want to add the same gradient to the next two sections. Instead of doing that, create a new class selector called `bb1-window`, and move the `height` and `background` properties and values from `.bb1a` to the new class selector.
```

Comment:
```suggestion
You want to add the same gradient to the next two sections. Instead of doing that, create a new class attribute called `bb1-window`, and move the `height` and `background` properties and values from `.bb1a` to the new class selector.
```

Reply:
Thank you for suggestion @Sboonny. I'm not sure but,
using selector feels more acceptable to me, because we're using it in style element.

Reply:
I agree - when we're writing CSS, it's a selector. When we're adding it to an HTML element, it's an attribute.

Reply:
[In the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors) class selector is used as you described, I am worried saying selector people will type ` <div .class="value>`, but hey if another moderator approve it, I will as well

Reply:
This is a CSS selector. We should use selector.